docker/environment.yml

```diff
@@ -22,11 +36,11 @@ dependencies:
   - descartes
   # Pin rasterio for ravenpy, remove on next build.
   # See https://github.com/CSHS-CWRA/RavenPy/commit/eae66e9afc30e2381e9119644a0695d1d248c739
-  - rasterio <= 1.2.6
-  - gdal  # for osgeo
-  - geopandas
-  - pandas
-  - rioxarray
+    # - rasterio <= 1.2.6  # from ravenpy
```

Comment:
```suggestion
    # - rasterio  # from ravenpy
```

Reply:
Done in commit https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/pull/95/commits/eaa2b01954a748b999b71c3ceeef115e6f714cd2